api/controllers/blobs/view.js

```diff
@@ -1,21 +1,21 @@
 module.exports = {
-  friendlyName: "TestFriendlyName",
-  description: "Test description",
+  friendlyName: 'TestFriendlyName',
+  description: 'Test description',
   inputs: {
 
   },
 
   exits: {
     success: {
-      outputDescription: "Test output description",
-      outputType: "ref",
+      outputDescription: 'Test output description',
+      outputType: 'ref',
     },
-    forbidden: { responseType: "forbidden" },
-    notFound: { responseType: "notFound" },
+    forbidden: { responseType: 'forbidden' },
+    notFound: { responseType: 'notFound' },
   },
 
   fn: async (inputs) => {
-    console.log("Viewing something");
+    console.log('Viewing something');
```

Comment:
Small change - but since the other console logs has Viewing all configs/source etc. we should also do the same for `Viewing all blobs`